test/propagation/grpc_propagator_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'helper'
+require 'ddtrace/tracer'
+require 'ddtrace/propagation/grpc_propagator'
+
+class GRPCPropagatorTest < Minitest::Test
```

Comment:
We've been working on moving away from Minitest towards RSpec. Does this test benefit from something in Minitest that isn't available or conducive to RSpec?

Reply:
No major benefit of Minitest over RSpec. I will rewrite this in RSpec instead